client/go/configyaml/parse.go

```diff
@@ -53,7 +54,9 @@ func MapToAny(in map[string]any, out any) error {
 		}
 		newMap[k] = v
 	}
-	yamlText, err := yaml.Marshal(newMap)
+	// Use JSON marshaling from the standard library because the YAML library is buggy.
```

Comment:
Suggest to mention "JSON is a subset of YAML" because we name the return value `yamlText`.

```suggestion
	// Use JSON marshaling from the standard library because the YAML library is buggy.
	// JSON is a subset of YAML, so we can use it as a reliable alternative.
```

Reply:
Done